src/compas/scene/sceneobject.py

```diff
@@ -82,13 +83,14 @@ class SceneObject(TreeNode):
 
     color = ColorAttribute()
 
-    def __new__(cls, item, **kwargs):
+    def __new__(cls, item=None, **kwargs):
         sceneobject_cls = get_sceneobject_cls(item, **kwargs)
         return super(SceneObject, cls).__new__(sceneobject_cls)
 
-    def __init__(self, item, name=None, color=None, opacity=1.0, show=True, frame=None, transformation=None, context=None, **kwargs):  # fmt: skip
-        # type: (compas.geometry.Geometry | compas.datastructures.Datastructure, str | None, compas.colors.Color | None, float, bool, compas.geometry.Frame | None, compas.geometry.Transformation | None, str | None, dict) -> None
-        name = name or item.name
+    def __init__(self, item=None, name=None, color=None, opacity=1.0, show=True, frame=None, transformation=None, context=None, **kwargs):  # fmt: skip
```

Comment:
is this to allow for the creation of a `SceneObject` without a data item? in that case, would it not make more sense to add something like `EmptySceneObject`?